torch/testing/_internal/common_quantization.py

```diff
@@ -176,6 +274,13 @@ def wrapper(*args, **kwargs):
             fn(*args, **kwargs)
     return wrapper
 
+try:
```

Comment:
thoughts on keeping the torchvision tests and utils in separate files?  It seems to be for a different use case (domain testing of correctness), and it would be useful for code readers to be able to ignore it unless they are specifically working on torchvision

Reply:
this is not really about torchvision I think. we are using torchvision because it is already quantized with eager mode. the goal is to do end to end testing.

Reply:
the name of the PR might be confusing, I'll change it

Reply:
One option is to keep this test as part of the torchvision repo

Reply:
yeah, my high level thoughts are:
1. it's definitely valuable to know that torchvision eager mode and FX produce the same results
2. I'm not sure if checking in a torchvision e2e test would be a net benefit, I'd be worried about this test being flaky and I'm not sure if depending on torchvision from unit tests is good
3. I think we can get the same guarantees with less testing (we can assume DDP is working correctly without testing it here, and we can assume that if all the building blocks and patterns of torchvision models work, then the models themselves would work).
4. I don't feel super strongly about (3) and I could be wrong as well, we could check in this code and if it leads to problems later we can delete it

Reply:
> the goal is to do end to end testing.

optional, but feels like e2e comparisons of small building blocks could give us the same guarantees, without the torchvision dependency

Reply:
fx is already depending on torchvision, I think it's fine: https://github.com/pytorch/pytorch/blob/master/test/test_fx.py#L10-L15

it's just easier to test torchvision directly. if we want to test sub blocks of torchvision models we will also need to write the test for them. we could of course spend time to do that, but I don't think it's a priority at the moment and not sure how valuable it is to remove the torchvision dependency